OpenEdX/RouteController.swift

```diff
@@ -49,7 +49,11 @@ class RouteController: UIViewController {
             present(navigation, animated: false)
         } else {
             let controller = UIHostingController(
-                rootView: SignInView(viewModel: diContainer.resolve(SignInViewModel.self)!)
+                rootView: SignInView(
+                    viewModel:
+                        diContainer.resolve(SignInViewModel.self,
+                                            argument: LogistrationSourceScreen.default)!
+                )
```

Comment:
Can we format it in some way? It looks strange.
<img width="639" alt="image" src="https://github.com/openedx/openedx-app-ios/assets/127732735/44ca4d6c-1a26-465d-9bd0-0eed4d215245">
->
```
 let controller = UIHostingController(
    rootView: SignInView(
        viewModel: diContainer.resolve(
            SignInViewModel.self,
            argument: LogistrationSourceScreen.default
        )!
    )
)
```